cmd/clear.go

```diff
@@ -14,7 +14,10 @@ func clearCommand() *cobra.Command {
 	clearCmd := &cobra.Command{
 		Use:   "clear [<path>]",
```

Comment:
Should we include the aliases created in https://github.com/puppetlabs/wash/blob/master/cmd/rootMain.go#L51-L59 in each of the command objects? That way, the help text prints those out too. It would also be nice if the command objects were the source of truth for their aliases on top of everything else.